mypy/test/data.py

```diff
@@ -249,7 +249,19 @@ def runtest(self) -> None:
             pytest.skip()
         suite = self.parent.obj()
         suite.setup()
-        suite.run_case(self)
+        try:
+            suite.run_case(self)
+        except Exception:
+            # As a debugging aid, support copying the contents of the tmp directory somewhere
+            target_dir = self.config.getoption('--save-failures-to', None)  # type: Optional[str]
+            print(target_dir)
```

Comment:
Is this a debug print? Maybe you want to print a nicely formatted error showing which test's temp dir was saved where instead?

Reply:
Made it do a nice print